plugins/lookup/k8s.py

```diff
@@ -183,15 +183,31 @@
         type: complex
 """
 
+import os
+
 from ansible.errors import AnsibleError
 from ansible.module_utils.common._collections_compat import KeysView
-from ansible.plugins.lookup import LookupBase
+from ansible.module_utils.common.validation import check_type_bool
 
 from ansible_collections.kubernetes.core.plugins.module_utils.common import (
     K8sAnsibleMixin,
     get_api_client,
 )
 
+try:
+    enable_turbo_mode = check_type_bool(os.environ.get("ENABLE_TURBO_MODE"))
```

Comment:
Could you please example in README.md or https://github.com/ansible-collections/kubernetes.core/blob/main/docs/ansible_turbo_mode.rst? So it would be easy for end-user to use.

Reply:
Thanks for the feedback.
README has been updated